gnuradio-runtime/lib/vmcircbuf_sysv_shm.cc

```diff
@@ -44,7 +44,7 @@ vmcircbuf_sysv_shm::vmcircbuf_sysv_shm(int size) : gr::vmcircbuf(size)
     int pagesize = gr::pagesize();
 
     if (size <= 0 || (size % pagesize) != 0) {
-        fprintf(stderr, "gr::vmcircbuf_sysv_shm: invalid size = %d\n", size);
+        GR_LOG_ERROR(d_logger, boost::format("invalid size = %d\n") % size);
```

Comment:
Some error messages here should lose their `\n`, right?

So here and below.

Reply:
oh, I missed that one. Thanks!